feat: added col breaks feature

StefanNedelchev opened the pull request:
<!-- Thanks for submitting a pull request! Please provide enough information so that others can review your pull request. The two fields below are mandatory. -->

## Summary

The library supports row breaks but doesn't support col breaks which is also a valid XLSX feature.

## Test plan

Added unit test and integration test for writing col breaks

```shell
Page Breaks
      ✓ adds multiple row breaks
      ✓ adds multiple col breaks
```

StefanNedelchev commented:
I think I'll need a help with this because although the library now correctly writes col breaks, I think it doesn't properly read them, so if the document has any, they will be lost. Apparently this needs to be covered with tests which will make it easier to detect and fix

StefanNedelchev commented on package.json:
```diff
@@ -111,7 +111,7 @@
     "tmp": "^0.2.0",
     "unzipper": "^0.10.11",
-    "uuid": "^8.3.0"
+    "uuid": "^9.0.0"
```
I remember last year uuid released v9 to address vulnerability issue so it's always good to keep up to date with it

zurmokeeper commented:
@StefanNedelchev That's really great, thanks for the PR, but I'm going to have to spend some time getting to grips with this PR first, will reply as soon as I make progress!

StefanNedelchev commented on lib/xlsx/xform/sheet/col-breaks-xform.js:
```diff
@@ -0,0 +1,39 @@
+
+const ListXform = require('../list-xform');
+
+class ColBreaksXform extends ListXform {
```
This class has been using `RowBreaksXform` as an example

StefanNedelchev commented on lib/doc/column.js:
```diff
@@ -201,6 +199,21 @@ class Column {
   }
 
+  // Page Breaks
+  addPageBreak(t, b) {
```
The function was copied from `row.js` and modified to work for col break

StefanNedelchev commented:
> @StefanNedelchev That's really great, thanks for the PR, but I'm going to have to spend some time getting to grips with this PR first, will reply as soon as I make progress!

No worries, take your time. I added some refinements and also added some comments for better explanation

zurmokeeper commented:
@StefanNedelchev Thank you very much for your contribution, it's been merged and a new version has been released.

You are welcome to continue to PR and improve the package together, I see that there are still a lot of issues and good PRs in exceljs, my plan is to finish the issues in the current project and then go to exceljs to see what else can be solved! Thanks again!

StefanNedelchev commented:
> @StefanNedelchev Thank you very much for your contribution, it's been merged and a new version has been released.
> 
> You are welcome to continue to PR and improve the package together, I see that there are still a lot of issues and good PRs in exceljs, my plan is to finish the issues in the current project and then go to exceljs to see what else can be solved! Thanks again!

Thanks, it was my pleasure. It's sad to see a decent free stylesheet library to get left behind so I'm happy to contribute to this revived project. Since the PR is merged I'll delete my branch and my fork (until next PR)

zurmokeeper commented:
Yes, you're right, it was my problem, I changed it in the next update.
